tests/python_functional/functional_tests/parsers/set-tag/test_set_tag.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
I would put this testcase under: `functional_tests/rewrite/` dir

Reply:
Fixed.